guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -88,16 +104,19 @@ To create a pull request do the following:
 
 1.  Fill out the form, and click **Create pull request** again to submit the pull request&mdash;that’s it!
 
-## Report an issue
-If you find a typo or erroneous information in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs Github repository.
+## Report an issue {#issues}
+If you find a typo or erroneous information in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs Github repository. New GitHub issues include questions and requests for information to help us write or update content. Fill in as much information as you can.
 
 <div class="bs-callout bs-callout-info" id="info" markdown="1">
-Check the [existing issues](https://github.com/magento/devdocs/issues){:target} on Github to see if someone has already reported the issue.
+Check the [existing issues](https://github.com/magento/devdocs/issues) on Github to see if someone has already reported the issue.
 </div>
 
-All topics on this site contain a link to the Issues panel on Github, which makes it easy to report issues on specific topics. Click the **Give us feedback** link at the top left of the page to open an issue for that page.
+You have a couple of options to enter an issue:
+* Have general feedback? Create an issue on [GitHub DevDocs](https://github.com/magento/devdocs/issues/new){:target="_blank"}.
+* Have feedback on a specific DevDocs page? Click the **Report an Issue** link at the top right of the page to report on the currently open topic.
+* Have a Community code contribution? Create an issue to [request DevDocs content](https://github.com/magento/devdocs/issues/new?template=COMMUNITY_ISSUE_TEMPLATE.md){:target="_blank"}.
 
-## Edit metadata  
+## Edit metadata
```

Comment:
In the table - "Defines which guide the file belongs to and which left-hand menu collection the file will show up in. What you put here does not affect the top navigation menu, which is controlled by the `_/includes/navigation.html` file."

Reply:
Check for spellings of GitHub across page.